kubectl-rook-ceph.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Rook Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eEuo pipefail
+
+usages() {
+  cat <<EOF
+Description:
+  'kubectl rook ceph' provides common management and troubleshooting tools for Rook-Ceph.
+
+Commands:
+  ceph - run any 'ceph' CLI command as given
+    args  - any 'ceph' CLI arg (e.g., status)
+    flags - any 'ceph' CLI flag (e.g., --format=json-pretty)
+
+Usage:
+  kubectl rook ceph [command] [command args...] [command flags...]
```

Comment:
Not sure the difference matters for args and flags?
```suggestion
  kubectl rook-ceph ceph [command] [args...]
```

Reply:
initial format suggested by @BlaineEXE

Reply:
Ok, let's discuss more. Maybe we try to show something similar to what the `ceph --help` shows